src/PortingAssistant.Client.Analysis/AnalysisHandler.cs

```diff
@@ -160,6 +161,12 @@ public class PortingAssistantAnalysisHandler : IPortingAssistantAnalysisHandler
 
                 return solutionAnalysisResult;
             }
+            catch (OutOfMemoryException e)
+            {
+                _logger.LogError("Analyze solution {0} with error {1}", solutionFilename, e);
+                MemoryUtils.LogMemoryConsumption(_logger);
```

Comment:
@flywanli , @huawenm apart from the regular error logs with exception stacktrace, I'm logging the memory consumption separately with `information` level log, will our ops script capture it and include it in our weekly report?